app/src/main/java/org/oppia/app/player/exploration/ExplorationActivity.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.app.player.exploration
+
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** The starting point for exploration*/
```

Comment:
Nit: Documentation should include ending punctuation. And, there should be a space before the terminating ``*/``.

Reply:
Resolved changed this comment to 
`/** The starting point for exploration. */`